sdk/confidentialledger/azure-security-confidentialledger/test-resources.json

```diff
@@ -5,50 +5,84 @@
 		"BaseName": {
 			"type": "string",
 			"metadata": {
-				"description": "Ledger Name"
+				"description": "Name of the test ledger"
 			}
 		},
-		"TestApplicationId": {
+		"location": {
 			"type": "string",
+			"defaultValue": "eastus",
 			"metadata": {
-				"description": "Oid of the user"
+				"description": "Location of the test ledger"
 			}
 		},
-		"location": {
+		"tenantId": {
+			"type": "string",
+			"metadata": {
+				"description": "Tenant ID for the test application"
+			}
+		},
+		"testApplicationId": {
+			"type": "string",
+			"metadata": {
+				"description": "Application client ID for the test application"
+			}
+		},
+		"testApplicationOid": {
 			"type": "string",
-			"defaultValue": "eastus",
 			"metadata": {
-				"description": "Location of the ledger."
+				"description": "Application object ID for the test application"
+			}
+		},
+		"testApplicationSecret": {
+			"type": "string",
+			"metadata": {
+				"description": "Application secret for the test application"
 			}
 		}
 	},
 	"functions": [],
 	"variables": {},
-	"resources": [{
-		"name": "[parameters('BaseName')]",
-		"type": "Microsoft.ConfidentialLedger/ledgers",
-		"apiVersion": "2020-12-01-preview",
-		"location": "[parameters('location')]",
-		"properties": {
-			"ledgerType": "Public",
-			"aadBasedSecurityPrincipals": [{
-				"principalId": "[parameters('TestApplicationId')]",
-				"ledgerRoleName": "Administrator"
-			}]
+	"resources": [
+		{
+			"name": "[parameters('BaseName')]",
+			"type": "Microsoft.ConfidentialLedger/ledgers",
+			"apiVersion": "2020-12-01-preview",
+			"location": "[parameters('location')]",
+			"properties": {
+				"ledgerType": "Public",
+				"aadBasedSecurityPrincipals": [
+					{
+						"principalId": "[parameters('TestApplicationId')]",
+						"ledgerRoleName": "Administrator"
+					}
+				]
+			}
 		}
-	}],
-    "outputs": {
-        "LEDGER_NAME": {
+	],
+	"outputs": {
+		"AZURE_TENANT_ID": {
             "type": "string",
-            "value": "[parameters('BaseName')]"
+            "value": "[parameters('tenantId')]"
         },
+        "AZURE_CLIENT_ID": {
+            "type": "string",
+            "value": "[parameters('testApplicationId')]"
+        },
+        "AZURE_CLIENT_SECRET": {
+            "type": "string",
+            "value": "[parameters('testApplicationSecret')]"
+        },
```

Comment:
As far as I know, you don't need to explicitly set these as outputs, they should be getting exported as environment variables with these names by default. You can leave it in for now if you really need to merge this ASAP though.